Gems/ScriptCanvasPhysics/Code/CMakeLists.txt

```
@@ -19,9 +19,9 @@ ly_add_target(
             Legacy::CryCommon
         PUBLIC
             Gem::ScriptCanvas.Extensions
-    AUTOGEN_RULES
-        *.ScriptCanvasFunction.xml,ScriptCanvasFunctionRegistry_Header.jinja,AutoGenFunctionRegistry.generated.h
-        *.ScriptCanvasFunction.xml,ScriptCanvasFunctionRegistry_Source.jinja,AutoGenFunctionRegistry.generated.cpp
+     AUTOGEN_RULES
```

This is no longer aligned anymore
```suggestion
    AUTOGEN_RULES
```

---

Still needed, this module has ScriptCanvas functions that need to use these rules

---

I meant the indentation is not aligned.

---

I must've been tired and read "This is no longer needed" instead of "aligned".